players/g1_player.py

```diff
@@ -319,7 +338,8 @@ def next_target(self, curr_loc: Tuple[float, float], goal: Point2D, conf: float)
 
             # Add adjacent points to heap
             reachable_points, goal_dists = self.numpy_adjacent_and_dist(
```

Comment:
providing a cache of map points that are in sand will make this run faster, like so:

```python
is_in_sand_trap(next_p, self.sand_trap_matlab_polys, cache=self.map_points_in_sand_trap)
```

Reply:
altered this function call in the code